src/3d/qgsraycastingutils_p.cpp

```diff
@@ -233,6 +234,22 @@ namespace QgsRayCastingUtils
     return intersection( b, ray( r ) );
   }
 
+  // copied from https://stackoverflow.com/questions/23975555/how-to-do-ray-plane-intersection
+  bool rayPlaneIntersection( const Ray3D &r, const Plane3D &plane, QVector3D &pt )
+  {
+    float denom = QVector3D::dotProduct( plane.normal, r.direction() );
+    if ( abs( denom ) > 0.0001f ) // your favorite epsilon
```

Comment:
@wonder-sk I'm getting a warning during the build: 
```
/QGIS/src/3d/qgsraycastingutils_p.cpp:241: warning: using integer absolute value function 'abs' when argument is of floating point type [-Wabsolute-value]
    if ( abs( denom ) > 0.0001f )
         ^
```
it also suggests to use srd::abs instead

Reply:
@elpaso Fixed in 045b88e75bf18ae4f83e09f2dc5d73aafec724e3